lib/netsuite/records/vendor_payment_apply_list.rb

```diff
@@ -0,0 +1,32 @@
+module NetSuite
+  module Records
+    class VendorPaymentApplyList
+      include Support::Fields
+      include Namespaces::TranPurch
+
+      fields :apply
+
+      def initialize(attributes = {})
+        initialize_from_attributes_hash(attributes)
+      end
+
+      def apply=(applies)
+        case applies
+          when Hash
+            self.applies << VendorPaymentApply.new(applies)
+          when Array
+            applies.each { |apply| self.applies << VendorPaymentApply.new(apply) }
```

Comment:
Yeah no sense in propagating shit code.  I fixed it.

On Sun, Mar 1, 2015 at 9:38 PM, Jerome Cornet <notifications@github.com>
wrote:

> In lib/netsuite/records/vendor_payment_apply_list.rb
> <https://github.com/prburke/netsuite/pull/4#discussion_r25576047>:
>
> > +    class VendorPaymentApplyList
> > +      include Support::Fields
> > +      include Namespaces::TranPurch
> > +
> > +      fields :apply
> > +
> > +      def initialize(attributes = {})
> > +        initialize_from_attributes_hash(attributes)
> > +      end
> > +
> > +      def apply=(applies)
> > +        case applies
> > +          when Hash
> > +            self.applies << VendorPaymentApply.new(applies)
> > +          when Array
> > +            applies.each { |apply| self.applies << VendorPaymentApply.new(apply) }
>
> mhhh. as long as you're not calling the method twice it works I guess :-(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prburke/netsuite/pull/4/files#r25576047>.
>